packages/shared/src/components/BookmarkFeedLayout.tsx

```diff
@@ -37,22 +38,23 @@ export default function BookmarkFeedLayout({
   const { user, tokenRefreshed } = useContext(AuthContext);
   const [showEmptyScreen, setShowEmptyScreen] = useState(false);
   const [showSharedBookmarks, setShowSharedBookmarks] = useState(false);
-
+  const defaultKey = generateQueryKey(RequestKey.Bookmarks, user);
   const feedProps = useMemo<FeedProps<unknown>>(() => {
     if (searchQuery) {
       return {
         feedName: 'search-bookmarks',
-        feedQueryKey: ['bookmarks', user?.id ?? 'anonymous', searchQuery],
+        feedQueryKey: defaultKey.concat(searchQuery),
```

Comment:
I find array spread clearer 
```suggestion
        feedQueryKey: [...defaultKey, ...searchQuery],
```

Reply:
Good point. I just think the former feels like reading an actual English sentence and making it intuitive. Though this is probably just a personal preference. WDYT? @rebelchris

Reply:
I personally also like spread more, but it's really optional.
Do think we tend to use spread more than concat already.